### PR TITLE
Update UserNSSandbox to include the `/dev/zero` fix

### PR DESCRIPTION
**Pull request by DilumAluthge**
This updates UserNSSandbox to include the `/dev/zero` fix from https://github.com/staticfloat/Sandbox.jl/pull/70

### CODE REVIEW

**Comment by DilumAluthge** [comment]
```
---> cd ${WORKSPACE}/srcdir/Sandbox.jl/deps
/bin/bash: line 34: cd: /workspace/srcdir/Sandbox.jl/deps: No such file or directory
```


Um..... https://github.com/staticfloat/Sandbox.jl/tree/ed87f079edb1cef60c32ad5fbfbcd38c70a1314a/deps

**Comment by DilumAluthge** [comment]
@giordano Any idea what I've done wrong?

**Comment by DilumAluthge** [comment]
Okay, it seems to be working now, although I don't understand why I needed to make this change on line 14:

```diff
- cd ${WORKSPACE}/srcdir/Sandbox.jl/deps
+ cd ${WORKSPACE}/srcdir/Sandbox.jl-cac5c095dfe27fa774bdad65957d45f476b220680c9a2038c0dd2c25cdd0b3e6/deps
```

File: U/UserNSSandbox/build_tarballs.jl [hunk 1]
```diff
@@ -1,17 +1,17 @@
...
 sources = [
     GitSource("https://github.com/staticfloat/Sandbox.jl",
-              "9bd66da515893f1997feb51d228bcb3dc1d68771"),
+              "ed87f079edb1cef60c32ad5fbfbcd38c70a1314a"),
 ]
 
 # Bash recipe for building across all platforms
 script = raw"""
-cd ${WORKSPACE}/srcdir/Sandbox.jl/deps
+cd ${WORKSPACE}/srcdir/Sandbox.jl-cac5c095dfe27fa774bdad65957d45f476b220680c9a2038c0dd2c25cdd0b3e6/deps
```
Comment (Keno) [inline U/UserNSSandbox/build_tarballs.jl:14]: `Sandbox.jl-*` to avoid having to update the script for the next version bump

Comment (DilumAluthge) [reply U/UserNSSandbox/build_tarballs.jl]: Good call, done!

**Comment by giordano** [comment]
> Okay, it seems to be working now, although I don't understand why I needed to make this change on line 14: 

The URL must end with `.git`: https://github.com/JuliaPackaging/Yggdrasil/pull/4289#discussion_r788552413

**Pull request merged**